modules/F50_base_aggregator.sh

```diff
@@ -133,9 +133,14 @@ output_details() {
     DATA=1
   fi
   if [[ "$S22_PHP_VULNS" -gt 0 ]]; then
```

Comment:
You need to check for the number of php ini issues. S22_PHP_VULNS is something different.